ampersand-multifield-view.js

```diff
@@ -39,19 +33,17 @@ var MultiFieldView = View.extend({
       throw new Error('must pass in a name');
     }
 
-    if (!this.fields) {
-      throw new Error('must have an array of fields');
-    }
-
-    this.value = spec.value || {};
+    this.beforeSubmit = spec.beforeSubmit || this.beforeSubmit;
+    this.fields = spec.fields || this.fields || [];
```

Comment:
don't we need to add `extraProperties: 'allow'` if we take `fields` out of `props`? ([extraProperties](http://ampersandjs.com/docs#ampersand-state-extraproperties))